test/e2e/tests/assets/asset-list.spec.ts

```diff
@@ -28,18 +30,11 @@ function buildFixtures(title: string, chainId: number = 1337) {
 }
 
 describe('Multichain Asset List', function (this: Suite) {
-  // Apply to all tests in this suite
-  before(function () {
-    if (!process.env.PORTFOLIO_VIEW) {
-      this.skip();
-    }
-  });
-
   it('persists the preferred asset list preference when changing networks', async function () {
     await withFixtures(
       buildFixtures(this.test?.fullTitle() as string),
       async ({ driver }: { driver: Driver }) => {
-        await loginWithBalanceValidation(driver);
+        await loginWithoutBalanceValidation(driver);
         const headerNavbar = new HeaderNavbar(driver);
```

Comment:
I'd strongly suggest to wait for the balance, before performing any step. This could cause the token controller state not being initialized yet.

You could do something like:

```
const homePage = new HomePage(driver);
await homePage.check_expectedBalanceIsDisplayed('24.9978', 'POL'); 
```

Improvement: instead of hardcoding the balance, you could maybe modify this method, to accept an optional param for the ticker (ie POL)

`check_localNodeBalanceIsDisplayed`